extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -397,6 +407,14 @@ export class SubscriptionService {
           .getEntitlements()
           .then(entitlement => {
             devAssert(entitlement, 'Entitlement is null');
+            if (this.checkEntitlementsAndEncryptionInvalid_(entitlement)) {
+              user().error(
+                TAG,
+                `${service['serviceId']}: Subscription granted and encryption enabled, ` +
```

Comment:
What's the `service['serviceId']` expression?

Reply:
That is grabbing the serviceId for the logging statement. I took this from line 388/398: if ((service['serviceId'] || 'local') == 'local')

Should I be using something else?